src/lib/util.ts

```diff
@@ -312,11 +305,24 @@ export async function getNetworkId(web3Provider?: Web3Provider): Promise<string>
 
       return await promise;
     }
-  } else if ((window as any).web3) {
-    const web3 = (window as any).web3;
-    return (await (web3.eth.net ? web3.eth.net.getId() : promisify(web3.version.getNetwork)())).toString();
   } else {
-    throw new Error("getNetworkId: unable to find web3");
+    let arc: Arc;
+
+    try {
+      arc = getArc();
+    } catch (ex) {
+      // Do nothing
```

Comment:
Why not `throw` or `console.log`?

Reply:
It is a normal occurrence that there may be no arc.  Just don't care.  We check for the case of no arc in the next line.